packages/salesforcedx-utils-vscode/src/helpers/utils.ts

```diff
@@ -14,11 +16,28 @@ export function isNullOrUndefined(object: any): object is null | undefined {
 }
 
 export function extractJsonObject(str: string): any {
-
   const jsonString = str.substring(
     str.indexOf('{'),
     str.lastIndexOf('}') + 1
   );
 
   return JSON.parse(jsonString);
 }
+
+// There's a bug in VS Code where, after a file has been renamed,
+// the URI that VS Code passes to the command is stale and is the
+// original URL.  See https://github.com/microsoft/vscode/issues/152993.
+//
+// To get around this, fs.realpathSync.native() is called to get the
+// URI with the actual file name.
+export function flushFilePath(filePath: string): string {
```

Comment:
Should we have the input here be of type `string | undefined` and error if an undefined value is passed?  That would save us a lot of `if(inputFile)` checks in the calling code.

Reply:
Done (this will be in my other PR, as I'm going to close this one out)